frontend/src/pages/quiz.tsx

```diff
@@ -46,15 +46,15 @@ const Quiz: FC = () => {
         <h1 className="mb-8 font-display text-2xl font-medium md:mb-12 md:rounded-3xl md:bg-gray-surface md:p-6 md:text-4xl md:text-primary-700">
           {t('landing.header')}
         </h1>
-        <h2 id="take-this-quiz" className="my-4 font-display text-xl font-medium">
+        <h2 id="take-this-quiz" className="font-display text-xl font-bold">
           {t('landing.p1')}
         </h2>
-        <p>{t('landing.p2')}</p>
-        <h2 id="what-you-ll-need" className="my-4 font-display text-xl font-medium">
+        <p className='my-8'>{t('landing.p2')}</p>
+        <h2 id="what-you-ll-need" className="my-4 font-display text-xl font-bold">
           {t('landing.whatyouneed')}
         </h2>
-        <p>{t('landing.p3')}</p>
-        <ul className="list-disc space-y-2 pl-10">
+        <p className='mb-0'>{t('landing.p3')}</p>
+        <ul className="list-disc pl-10">
```

Comment:
Paragraph and list style should be consitent across the application.

```suggestion
        <p>{t('landing.p3')}</p>
        <ul className="list-disc space-y-2 pl-10">
```

Reply:
I've looked other lists and only one page is using `space-y-1` and it's `rules-of-thumb-for-public-pensions.tsx`. I'll fixed it to match other lists.